osbrain/proxy.py

```diff
@@ -413,4 +414,7 @@ def shutdown(self, timeout=10.):
             Timeout, in seconds, to wait for the agents to shutdown.
         """
         self.shutdown_agents(timeout)
-        super()._pyroInvoke('daemon_shutdown', (), {}, flags=0)
+        try:
+            super()._pyroInvoke('daemon_shutdown', (), {}, flags=0)
```

Comment:
Aside from catching the exception, maybe we could also use `Pyro4.FLAGS_ONEWAY` here. What do you think?

Reply:
It would change the current behavior: to block if the name server is busy and until it actually shuts down. We could consider this if we find out it is a better default behavior, but for now I think it is better if we do not change that.